.github/workflows/ci.yml

```diff
@@ -274,3 +273,26 @@ jobs :
 
       - name : "Build Benchmark Project"
         run : ./gradlew :benchmark:app:assemble
+
+  all-checks:
+    if: always()
+    runs-on: ubuntu-latest
+    needs:
+      - test-ubuntu
+      - test-windows
+      - ktlint
+      - lint
+      - publish-maven-local
+      - publish-snapshot
+      - test-gradle-plugin
+      - kapt-for-dagger-factories
+      - instrumentation-tests
+      - gradle-wrapper-validation
+      - build-benchmark-project
+
+    steps:
+      - name: require that all other jobs have passed
+        uses: re-actors/alls-green@release/v1
+        with:
+          allowed-skips: publish-snapshot
```

Comment:
Nice, I love that this is explicit. I was about to leave a comment on how we handle skips before seeing it 😁